### PR TITLE
Couple of unit-tests for the logbook resource and tidy up

### PR DESCRIPTION
**Pull request by algairim**
Signed-off-by: Mykola Mandra <mykola.mandra@cloudsoftcorp.com>

### CODE REVIEW

File: rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java [hunk 1]
```diff
@@ -38,10 +38,13 @@
 
     @Override
     public Response logbookQuery(HttpServletRequest request, LogBookQueryParams params) {
-        // TODO discuss about if is ok or not allow only to the root user to see the logs.
```
Comment (algairim) [inline rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java:41]: Other than root user is out of scope for now.

Comment (algairim) [reply rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java:41]: cc @jcabrerizo

File: core/src/main/java/org/apache/brooklyn/util/core/logbook/DelegatingLogStore.java [hunk 2]
```diff
@@ -66,7 +66,8 @@ private LogStore loadDelegate() {
             Class<? extends LogStore> clazz = (Class<? extends LogStore>) clu.loadClass(className);
             delegate = createLogStoreProviderInstance(mgmt, clazz);
         } catch (Exception e) {
-            log.warn("Brooklyn Logbook: unable to instantiate Log Store " + className, ". Forcing to use FileLogStore", e);
+            log.warn("Brooklyn Logbook: unable to instantiate Log Store '" + className + "'. Fall back on FileLogStore", e.getMessage());
+            log.debug("", e);
```
Comment (algairim) [inline core/src/main/java/org/apache/brooklyn/util/core/logbook/DelegatingLogStore.java:70]: Print full stack trace at debug log level only.

File: rest/rest-resources/src/test/java/org/apache/brooklyn/rest/testing/BrooklynRestApiTest.java [hunk 3]
```diff
@@ -143,14 +143,18 @@ private TestShutdownHandler createShutdownHandler() {
         return new TestShutdownHandler();
     }
 
+    protected BrooklynProperties getBrooklynProperties() {
+        return null;
+    }
```
Comment (algairim) [inline rest/rest-resources/src/test/java/org/apache/brooklyn/rest/testing/BrooklynRestApiTest.java:148]: This allows to introduce Brooklyn properties at instantiation stage.

File: rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java (hunk 1 above)
Comment (jcabrerizo) [reply rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java:41]: Good to me @algairim. Thanks

**Comment by jcabrerizo** [comment]
This looks good to me @algairim, happy to merge once the tests pass

File: rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java (hunk 1 above)
Comment (algairim) [reply rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LogbookResource.java:41]: @jcabrerizo , I tagged you to keep these changes in your records. Please do not waste your personal time on this review :)

**Pull request merged**